playgrounds/html/setup.js

```diff
@@ -0,0 +1,217 @@
+/* eslint camelcase: ["off"]*/
+/* eslint import/no-commonjs: ["off"]*/
+
+const { MeiliSearch } = require('meilisearch')
+
+;(async () => {
+  const client = new MeiliSearch({
+    host: 'http://127.0.0.1:7700',
+    apiKey: 'masterKey',
+  })
+
+  const index = client.index('docs')
+
+  const dataset = [
```

Comment:
I'm not sure what kind of document it is.

Reply:
What do you mean?